plugin-server/src/cdp/consumers/cdp-cyclotron-worker-hogflow.consumer.ts

```diff
@@ -32,6 +32,16 @@ export class CdpCyclotronWorkerHogFlow extends CdpCyclotronWorker {
                     return null
                 }
 
+                if (hogFlow.status === 'paused' || hogFlow.status === 'stopped') {
```

Comment:
Whats the distinction between paused and stopped?

Feels like we would just need one of these or maybe something more generic like `inactive`

Also this logic isn't necessary as its covered in the hogFlowManager here https://github.com/PostHog/posthog/blob/master/plugin-server/src/cdp/services/hogflow-manager.service.ts#L138